### ISSUE TITLE
Bug with files with non-ASCII names and insides

### ISSUE THREAD

**Issue by rilian-la-te** [cuckoosandbox/cuckoo#1690]
Python simply crashes with non-English folder or file names in guest with:
```
2017-07-03 18:49:42,640 [analyzer] ERROR: Pipe command handler exception occurred (command FILE_NEW args 'C:\\Documents and Settings\\Admin\\Application Data\\Microsoft\\\xd0\xa8\xd0\xb0\xd0\xb1\xd0\xbb\xd0\xbe\xd0\xbd\xd1\x8b\\~$Normal.dotm').
Traceback (most recent call last):
  File "C:\tmp8cgwbs\analyzer.py", line 403, in dispatch
    response = fn(arguments)
  File "C:\tmp8cgwbs\analyzer.py", line 311, in _handle_file_new
    self.analyzer.files.add_file(data.decode("utf8"), self.pid)
  File "C:\tmp8cgwbs\analyzer.py", line 65, in add_file
    pid, filepath
  File "C:\Python27\lib\logging\__init__.py", line 1165, in info
    self._log(INFO, msg, args, **kwargs)
  File "C:\Python27\lib\logging\__init__.py", line 1284, in _log
    self.handle(record)
  File "C:\Python27\lib\logging\__init__.py", line 1294, in handle
    self.callHandlers(record)
  File "C:\Python27\lib\logging\__init__.py", line 1334, in callHandlers
    hdlr.handle(record)
  File "C:\Python27\lib\logging\__init__.py", line 757, in handle
    self.emit(record)
  File "C:\tmp8cgwbs\lib\common\results.py", line 102, in emit
    self.send("{0}\n".format(msg))
UnicodeEncodeError: 'ascii' codec can't encode characters in position 146-152: ordinal not in range(128)
```

**Comment by jbremer** [comment]
Which version of Cuckoo are you running? From what I can tell this is a duplicate of #1606 which has been fixed in 5c7088ca82bbf1a3ed9c975c44584f1b32ec90f4 - for now you can upgrade to our latest alpha version (i.e., `pip install cuckoo==2.0.4a3`) which includes this fix. Let me know if that resolves your issue.

**Comment by rilian-la-te** [comment]
AFAIK, yes. But I will need to port modifications.

**Comment by jbremer** [comment]
Closing issue as the fix for this has been pushed in the `2.0.4` release.
